tox.ini

```diff
@@ -10,4 +10,4 @@ deps =
     -rrequirements.txt
     pytest-cov
     pycurl
-commands = py.test --cov --cov-config=.coveragerc tests/robottelo
+commands = py.test --cov --cov-config=.coveragerc tests/robottelo -m "not dummy"
```

Comment:
I'm not a fan of this pattern, having a marker used for the unit tests that we then have to explicitly exclude from running the unit tests.

Is there really harm to the 'dummy' test running during a normal unit test execution? Just the time to execute it?

It does feel like overkill for this use case, but I much prefer marker implementations that follow a pattern of modifying test collection unless an option is included to run them when using a 'negative' marker like this. In other words, a test marked with `dummy` does not run in normal test collection, and is only included in collection when `--dummy` is passed.

Reply:
Not a huge fan of this solution either. If you're okay with running the "empty" test, I'm more than happy to remove the marker.

Another possible approach could be to actually store the dummy test definition in a non-py plaintext file (or multiline string) and dump it to a `test_foo.py` file as a part of the fixture; and remove it for a cleanup.

Reply:
I think I like that second idea, you should be able to use `tempfile.NamedTemporaryFile` in a context, and on linux the `pytest.main` call should be able to read the written file. That will cleanly handle creation, potential name conflicts, and deletion on the context close.

Reply:
Looks like this mark isn't needed anymore for tox collection, with your updates to the unit test?

Reply:
ah, sorry